debian/rules

```diff
@@ -41,6 +42,7 @@ override_dh_auto_configure:
 
 override_dh_auto_test-arch:
 ifeq (,$(filter nocheck,$(DEB_BUILD_OPTIONS)))
+	chmod +x tests/test-pull-bareuseronly.sh
```

Comment:
Why is this needed? `test-pull-bareuseronly.sh` is mode 0755 in git, and this apparently isn’t needed for any of the other tests.

Reply:
This came from debian. Possibly it's not 0755 in the `make dist` tarball. Let me check.

Reply:
That looks like it can be dropped now - https://salsa.debian.org/debian/ostree/commit/022fab56dc536c07667e89d63f836ad05c013af7. He'd added it through a patch.